modules/local/replace_dots.nf

```diff
@@ -8,35 +8,37 @@ process REPLACE_DOTS {
     'docker.io/ubuntu:20.04' }"
 
     input:
-    tuple val(meta), path(file)
+    tuple val( meta ), path( file )
 
     output:
     tuple val( meta ), file( "*bed" ),  emit: bed
     path "versions.yml"              ,  emit: versions
 
-    when:
-    task.ext.when == null || task.ext.when
-
-    shell:
-    def prefix  = task.ext.prefix ?: "${meta.id}"
+    script:
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def DOTS_VERSION = "2.0"
     def VERSION = "9.1" // WARN: Version information not provided by tool on CLI. Please update this string when bumping container versions.
-    $/
-    cat "${file}" | sed 's/\./0/g' > "${prefix}_nodot.bed"
+    """
+    cat "${file}" \\
```

Comment:
Seem to be a lot of local modules with this pattern: cat a file and modify with sed or awk (or both). Not a job for this PR but could they be replaced with a single configurable module - I use the nf-core GAWK module a lot in metagenomeassembly and it does a lot of this basic file manipulation without re-inventing the wheel multiple times

Reply:
All but 2 `cat > sed` modules have been re-written into GAWK: GETMINMAXPUNCHES ( has two outputs ) and  GENERATE_GENOME_FILE (requires sorting after awk).

Reply:
If you love awk you can pipe to sort directly inside an awk program: https://unix.stackexchange.com/a/560190